src/js/controller/controller.js

```diff
@@ -503,7 +514,7 @@ Object.assign(Controller.prototype, {
                     mediaPool.prime();
                 }
 
-                if (playReason === 'playlist') {
+                if (playReason === 'playlist' && _model.get('autoPause').viewability) {
```

Comment:
is _model.get('autoPause') always defined and an object?

Reply:
Yup, we default it to:

```json
autoPause: {
  viewability: false
}
```
https://github.com/jwplayer/jwplayer/blob/master/src/js/api/config.js#L13